scripts/initialize-citus.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -e
+
+DATABASE=commcarehq_citus
+
+function exec_sql() {
+    HOST=$1
+    DB=$2
+    SQL=$3
+    docker exec -it $HOST psql -U postgres $DB -c "$SQL"
+}
+
+function get_worker_status() {
+    echo "`docker inspect -f {{.State.Health.Status}} citus_worker_1`"
+}
+
+COMPOSE_PROJECT_NAME=citus docker-compose -f docker/citus-compose.yml up -d
```

Comment:
Why is this being setup in a separate docker compose rather than being added to the existing compose cluster? Will it be possible to control machines in this cluster using `./scripts/docker`?

Reply:
I had some trouble getting it to run when adding the services to our compose files. I also thought that this is only for ICDS so I didn't want to force everyone to run Citus.